core/common/src/main/java/alluxio/underfs/ObjectUnderFileSystem.java

```diff
@@ -549,6 +564,27 @@ protected String convertToFolderName(String key) {
   }
 
   /**
+   * Owner of the mounted bucket.
+   *
+   * @return owner for the file
+   */
+  protected abstract String getBucketOwner();
+
+  /**
+   * Group of the mounted bucket.
+   *
+   * @return group for the file
+   */
+  protected abstract String getBucketGroup();
+
+  /**
+   * Mode of the mounted bucket.
+   *
+   * @return owner for the file
+   */
+  protected abstract short getBucketMode();
```

Comment:
Perhaps consolidate this into `getPermissions`? Although all the current implementations are cheap, there may be an Object Storage which needs to make an outgoing request to gather this information and we never use any one part independently